tests/openthread/Makefile

```diff
@@ -11,43 +12,50 @@ RIOTBASE ?= $(CURDIR)/../..
 # Comment this out to disable code in RIOT that does safety checking
 # which is not needed in a production environment but helps in the
 # development process:
-CFLAGS += -DDEVELHELP -Wall
+#CFLAGS += -DDEVELHELP -Wall
 
 # Change this to 0 show compiler invocation lines by default:
 QUIET ?= 1
 
 USEPKG += openthread
-USEMODULE += openthread_contrib
-USEMODULE += libmbedcrypto
-USEMODULE += libopenthread
-USEMODULE += libopenthread-cli
-USEMODULE += xtimer
+OPENTHREAD_TYPE ?= ftd
+ifeq ($(OPENTHREAD_TYPE),mtd)
+    # MTD: A Minimal Thread Device does not have router functionality compiled in.
+    # As a result, it is not necessary to configure the routerrole on an MTD. At the same time, an MTD may or may not be sleepy.
```

Comment:
I'll do it but today, computer screen are much larger 120 bytes is good also ;-)